validator/valnode/valnode.go

```diff
@@ -36,32 +37,35 @@ var DefaultWasmConfig = WasmConfig{
 }
 
 type Config struct {
-	UseJit     bool                               `koanf:"use-jit"`
-	ApiAuth    bool                               `koanf:"api-auth"`
-	ApiPublic  bool                               `koanf:"api-public"`
-	Arbitrator server_arb.ArbitratorSpawnerConfig `koanf:"arbitrator" reload:"hot"`
-	Jit        server_jit.JitSpawnerConfig        `koanf:"jit" reload:"hot"`
-	Wasm       WasmConfig                         `koanf:"wasm"`
+	UseJit          bool                               `koanf:"use-jit"`
+	ApiAuth         bool                               `koanf:"api-auth"`
+	ApiPublic       bool                               `koanf:"api-public"`
+	Arbitrator      server_arb.ArbitratorSpawnerConfig `koanf:"arbitrator" reload:"hot"`
+	RedisExecRunner redis.ValidationServerConfig       `koanf:"redis-exec-runner"`
```

Comment:
Do we want a separate BOLD execution config?
I think we can use the existing config in arbConfigFetcher().RedisValidationServerConfig
There is a separate queue for BOLD with the same module root, so it makes sense to me that we only need one config, and if we really don't want BOLD we don't need to have a queue for it. This has the advantage of the same config being used in normal operations (validation queue) as in emergencies.
Not 100% certain, though.